birdhouse/docker-compose.yml

```diff
@@ -1,3 +1,7 @@
 version: "3.4"
 
 services: {} # no services loaded by default
+
+networks:
+  default:
+    name: ${BIRDHOUSE_DEFAULT_NETWORK}
```

Comment:
Isn't this the default network name?  Do we still need to specify it?  But I guess no harm.

Reply:
Yes, the default value for this is the same as the default that docker-compose specifies. But, it's nice to be explicit about the name so that we don't have hard-coded values in the code with the assumption that the network name will never change.